Tests/Foundation/TestThread.swift

```diff
@@ -104,6 +104,8 @@ class TestThread : XCTestCase {
     func test_callStackSymbols() throws {
         #if os(Android) || os(OpenBSD)
         throw XCTSkip("Android/OpenBSD doesn't support backtraces at the moment.")
+        #elseif os(Windows)
+        throw XCTSkip("This test is unexpectedly crashing in CI at the moment.")
```

Comment:
I'm not certain why, but these tests were crashing in CI. Disabling for now until we can followup with a fix